### DISCUSSION TITLE
How to implement agendas on Evrete?

### DISCUSSION THREAD

**Discussion by Lati2211** [evrete/evrete#53] [Q&A]
Hi,I am trying to implement drools rule into Evrete .How do we create agendas in evrete?If you have a large number of rule based in different agendas, How can you call these rules on the basis of agendas?

**Comment by andbi** [comment]
There's no direct alternative to Drools agenda groups in Evrete. However, you can reproduce their behavior by providing a custom implementation of Activation Manager as described in https://evrete.org/docs/conflict-resolution/
